### DISCUSSION TITLE
Is ebiten suitable to build a point and click adventure

### DISCUSSION THREAD

**Discussion by vbd** [hajimehoshi/ebiten#2980] [Q&A] [answered]
Is ebiten a good choice to build a point and click adventure in the style of Sierra or LucasArts? Are there already any examples out there?

**Comment by tinne26** [comment, marked as answer]
Yes, it's totally capable of that, and I'd say a reasonable choice for it. I don't know of any game like that made with Ebitengine, but if you look at the commercial games on the [showcase page](https://ebitengine.org/en/showcase.html) you will notice most of those already do more complicated things than what you usually need for point and click games.

**Reply by vbd** [reply]
Thank you. So we will try to use it for some kind of storytelling for internal training.
